umbra-js/hardhat.config.ts

```diff
@@ -0,0 +1,102 @@
+import { config as dotenvConfig } from 'dotenv';
+import { resolve } from 'path';
+dotenvConfig({ path: resolve(__dirname, './.env') });
+
+import { HardhatUserConfig } from 'hardhat/config';
+import { NetworkUserConfig } from 'hardhat/types';
+
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-waffle';
+
+const chainIds = {
+  ganache: 1337,
+  goerli: 5,
+  hardhat: 1337,
+  kovan: 42,
+  mainnet: 1,
+  rinkeby: 4,
+  ropsten: 3,
+};
+
+// Ensure that we have all the environment variables we need.
+const mnemonic = 'test test test test test test test test test test test junk';
+// if (!process.env.MNEMONIC) {
+//   console.warn('Please set your MNEMONIC in a .env file');
+// } else {
+//   mnemonic = process.env.MNEMONIC;
+// }
+
+let infuraApiKey = '';
+if (!process.env.INFURA_ID) {
+  console.warn('Please set your INFURA_API_KEY in a .env file');
+} else {
+  infuraApiKey = process.env.INFURA_ID;
+}
```

Comment:
Nit: but we can condense this a bunch as shown / as we do in the `/contracts/hardhat.config.ts` file

```suggestion
const infuraApiKey = process.env.INFURA_API_KEY;
if (!infuraApiKey) throw new Error('Please set your INFURA_API_KEY in a .env file');
```